### ISSUE TITLE
The bot doesn't play anything

### ISSUE THREAD

**Issue by 2duviz2** [Raptor123471/DingoLingo#126]
The bot works, but when I try to play something, the bot download the music, send the message and doesn't play anything, and when I put $queue the bot says that the queue is empty

**Comment by solaluset** [comment]
Do you have any output in the console?

**Comment by 2duviz2** [comment]
no, literally anything, the bot only download the song, and send the embed, the output only says video downloader stuff, but there is no error

**Comment by solaluset** [comment]
Can you share the link which causes this?
What platform are you running on?
If Linux/MacOS, do you have ffmpeg and necessary libraries installed?

**Comment by EdwinKorner** [comment]
I have the same problem. I'm using windows, all of the logs it says everything is fine and the bot sends all the information and joins the call but nothing plays. Maybe it has something to do with the version of the packages?  What version should everything be?

![Screenshot 2023-05-17 202616](https://github.com/Raptor123471/DingoLingo/assets/90620059/279cbc99-2645-405d-840b-e4d14ce77bc6)
![Screenshot 2023-05-17 202730](https://github.com/Raptor123471/DingoLingo/assets/90620059/30b635c0-6ef7-4522-9e85-225f657d4105)
